Dockerfile

```diff
@@ -19,12 +19,20 @@ ENV PATH=${PYBIN}:${PATH} \
 RUN ln -s /opt/python/cp${PYV}* /opt/python/v
 
 RUN pip install future numpy setuptools wheel && \
-    pip install tensorflow-gpu==1.7 && \
-    pip install tensorflow-gpu==1.11 --target /tensorflow/1_11 && \
-    pip install tensorflow-gpu==1.12rc2 --target /tensorflow/1_12 && \
-    pip install tf-nightly-gpu --target /tensorflow/nightly && \
     rm -rf /root/.cache/pip/
 
+RUN if [ ${PYV} != "37" ] ; then \
+        pip install tensorflow-gpu==1.7 && \
+        pip install tensorflow-gpu==1.11 --target /tensorflow/1_11 && \
+        pip install tensorflow-gpu==1.12rc2 --target /tensorflow/1_12 && \
+        pip install tf-nightly-gpu --target /tensorflow/nightly && \
+        rm -rf /root/.cache/pip/; \
+    else \
+        # only nightly buidl of TF supports python 3.7 at that time
+        pip install tf-nightly-gpu && \
+        rm -rf /root/.cache/pip/; \
```

Comment:
Nitpicking: you can factor out `rm -rf /root/.cache/pip/;`

Reply:
It would be cleaner but will create another Docker layer we don't need, and previous one would be bigger - https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers.

Reply:
@JanuszL 
```
RUN if [ ${PYV} != "37" ] ; then \
        pip install tensorflow-gpu==1.7 && \
        pip install tensorflow-gpu==1.11 --target /tensorflow/1_11 && \
        pip install tensorflow-gpu==1.12rc2 --target /tensorflow/1_12 && \
        pip install tf-nightly-gpu --target /tensorflow/nightly; \
    else \
        # only nightly buidl of TF supports python 3.7 at that time
        pip install tf-nightly-gpu; \
      fi \
     && rm -rf /root/.cache/pip/; \
```
I was thinking about this.. this would not create an additional layer, would it?